examples/tutorials/Part X - Federated Learning with Secure Aggregation.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<p>is mometum supported? If not: better to remove this option from the notebook otherwise people will try to use it and get errors.</p>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2178/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Nice catch!</p>